test/e2e/data/templates/cluster-template-md-remediation.yaml

```diff
@@ -0,0 +1,261 @@
+                value: "${IBMPOWERVS_VIP_CIDR}"
+              - name: cp_enable
+                value: "true"
+              - name: cp_namespace
+                value: kube-system
+              - name: vip_ddns
+                value: "false"
+              - name: svc_enable
+                value: "true"
+              - name: vip_leaderelection
+                value: "true"
+              - name: vip_leaseduration
+                value: "5"
+              - name: vip_renewdeadline
+                value: "3"
+              - name: vip_retryperiod
+                value: "1"
+              - name: address
+                value: "${IBMPOWERVS_VIP}"
+              image: ghcr.io/kube-vip/kube-vip:v0.4.4
+              imagePullPolicy: Always
+              name: kube-vip
+              resources: {}
+              securityContext:
+                capabilities:
+                  add:
+                  - NET_ADMIN
+                  - NET_RAW
+              volumeMounts:
+              - mountPath: /etc/kubernetes/admin.conf
+                name: kubeconfig
+            hostAliases:
+            - hostnames:
+              - kubernetes
+              ip: 127.0.0.1
+            hostNetwork: true
+            volumes:
+            - hostPath:
+                path: /etc/kubernetes/admin.conf
+              name: kubeconfig
+          status: {}
+        owner: "root:root"
+        permissions: "0744"
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: IBMPowerVSMachineTemplate
+metadata:
+  name: "${CLUSTER_NAME}-control-plane"
+spec:
+  template:
+    spec:
+      serviceInstanceID: "${IBMPOWERVS_SERVICE_INSTANCE_ID}"
+      sshKey: "${IBMPOWERVS_SSHKEY_NAME}"
+      image:
+        name: "${IBMPOWERVS_IMAGE_NAME}"
+      network:
+        name: "${IBMPOWERVS_NETWORK_NAME}"
+      memory: ${IBMPOWERVS_CONTROL_PLANE_MEMORY:="4"}
+      processors: ${IBMPOWERVS_CONTROL_PLANE_PROCESSORS:="0.25"}
+      sysType: ${IBMPOWERVS_CONTROL_PLANE_SYSTYPE:="s922"}
+      procType: ${IBMPOWERVS_CONTROL_PLANE_PROCTYPE:="shared"}
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineDeployment
+metadata:
+  name: "${CLUSTER_NAME}-md-0"
+spec:
+  clusterName: "${CLUSTER_NAME}"
+  replicas: ${WORKER_MACHINE_COUNT}
+  selector: {}
+  template:
+    metadata:
+      labels:
+        e2e.remediation.label: ""
+    spec:
+      clusterName: "${CLUSTER_NAME}"
+      version: "${KUBERNETES_VERSION}"
+      bootstrap:
+        configRef:
+          name: "${CLUSTER_NAME}-md-0"
+          apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+          kind: KubeadmConfigTemplate
+      infrastructureRef:
+        name: "${CLUSTER_NAME}-md-0"
+        apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+        kind: IBMPowerVSMachineTemplate
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: "${CLUSTER_NAME}-mhc-0"
+spec:
+  clusterName: "${CLUSTER_NAME}"
+  maxUnhealthy: 100%
+  nodeStartupTimeout: 20m
```

Comment:
Are we able to match this number for the node boot up?

Reply:
For now we can stick to 20mins considering (Machine Build + Active state + Cluster join) phases, can update if needed as per observations for e2e results.